.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,62 @@
+name: Deployment
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+
+jobs:
+  compile:
+    runs-on: ubuntu-latest
+    name: Compile
+    strategy:
+      matrix:
+        value: ["ethereum-bridged-carbon", "polygon-bridged-carbon"]
+    steps:
+    - name: Check out source repository
+      uses: actions/checkout@v2
+    - name: Setup Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - name: Install The Graph CLI
+      run: npm install -g @graphprotocol/graph-cli
```

Comment:
Why are we installing the latest version of `graph-cli` here and then running another one to pick up `graph-ts`? Is it for the global flag?

We can also safely change those to `devDependencies` and npm should be able to run any needed scripts.

Reply:
No particular reason, I just copy-pasta'd some of this and never noticed the duplicate install

Reply:
Might be useful to swap the package files over to be dev dependencies since not each subgraph is using the same version of graph-cli/graph-ts